src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -1050,23 +1169,6 @@ public bool IsTopMostNode
             }
         }
 
-        internal void ConnectInput(int inputData, int outputData, NodeModel node)
```

Comment:
@mjkkirschner I will do that test. I would very much prefer that these were not caches. If it means that we have to turn off update events while the calculating is occurring and while frozen is being set, then we should do that.